ugvc/pipelines/lpr/train_lib_prep_recalibration_model.py

```diff
@@ -31,49 +33,103 @@ def run(argv):
     args = parser.parse_args(argv[1:])
     print(args)
 
+    nb_dir = args.notebooks_dir
     out_dir = args.out_dir
     os.makedirs(out_dir, exist_ok=True)
 
     featuremap_vcf, unfiltered_calls = localize_inputs(args)
 
     tp_args = [f"{out_dir}/estimated_tps.{suffix}" for suffix in ("bed", "vcf.gz", "parquet")]
     fp_args = [f"{out_dir}/estimated_fps.{suffix}" for suffix in ("bed", "vcf.gz", "parquet")]
-    tp_args.extend([args.tp_min_af, 1, args.tp_min_qual])
-    fp_args.extend([0, args.fp_max_af, args.fp_min_qual])
+    tp_args.extend([args.tp_min_af, 1])
+    fp_args.extend([0, args.fp_max_af])
     afs_bed = f"{out_dir}/afs.bed"
+    labeled_training_set = f"{out_dir}/labeled_featuremap_training_set.parquet"
 
+    # calibrate model on pass-filter events
     if args.calls_vcf:
         calls_vcf = f"{out_dir}/calls.snps.pass.vcf.gz"
-        featuremap_on_calls = f"{out_dir}/featuremap_on_calls.vcf.gz"
+        calls_parquet = f"{out_dir}/calls.snps.pass.parquet"
+        featuremap_on_calls_vcf = f"{out_dir}/featuremap_on_calls.vcf.gz"
+        featuremap_on_calls_parquet = f"{out_dir}/featuremap_on_calls.parquet"
+        qual_af_clusters_on_calls = f"{out_dir}/qual_af_clusters_on_calls.parquet"
         sp.print_and_run(f"bcftools view --type snps -f PASS -m2 -M2 {unfiltered_calls} -Oz -o {calls_vcf}")
         sp.print_and_run(
             f"bedtools intersect -a {featuremap_vcf} -b {calls_vcf} -header | "
```

Comment:
Does "bedtools intersect" make sure the alts in both vcfs match?

Reply:
Of course it doesn't.
I am planning to refine this

Reply:
Actually it makes no difference at all, since we later use the df merge to select the alleles.
This is just preliminary filtering for efficiency.

However I do classify variants as true/false in the end by position (if we have GTR), which should be refined.